pyext/src/analysis_trajectories.py

```diff
@@ -456,6 +504,21 @@ def read_stats_detailed(self,
             MSL_sum = pd.Series(self.add_restraint_type(DF, 'MSL_'))
             DF = DF.assign(MSL_sum=MSL_sum.values)
     
+        # if some score only type custom restraints defined by the user 
+        # need to be summed. Because of the do_sum flag available while
+        # calling the set_analyze_score_only_restraint() method on the 
+        # user defined restraint, they don't need to separately handled
+        # in read_traj_info. If you don't want the sum to be analyzed, just
+        # don't ever compute it by setting do_sum = False. 
+        if self.score_only_restraint:
+            for key, do_sum in self.sum_score_only_restraint.items():
+                prefix = key + "_"
+                all_restraint = [res for res in score_names if prefix in res]
+                if len(all_restraint)>1 and do_sum:
```

Comment:
flake8 will complain that you don't have spaces around '>' here (and probably elsewhere too).

Reply:
fixed.